src/engines/html5/media-source/base-media-source-adapter.js

```diff
@@ -251,10 +266,6 @@ export default class BaseMediaSourceAdapter extends FakeEventTarget implements I
    * @public
    */
   get currentTime(): number {
-    if (this.isLive()) {
-      const ct = this._videoElement.currentTime - this.getStartTimeOfDvrWindow();
-      return ct < 0 ? 0 : ct;
-    }
     return this._videoElement.currentTime;
```

Comment:
There is a diff between hlsjs and shaka in the video.currentTime
Need to align the response using the id3tags

Reply:
this is not the absolute time